### ISSUE TITLE
Deprecate functionalities related to using `token.pickle` and `credentials.json` in schematic 

### ISSUE THREAD

**Issue by linglp** [Sage-Bionetworks/schematic#1029]
**Is your feature request related to a problem? Please describe.**
After #1010 , we would want to use `service_account_creds` for google api authentication. There are couple of changes that need to happen for schematic: 
- Deprecate `schematic init --auth`. The `--auth` flag currently could let users choose `OOB` authentication or `service_account_creds`. In the future, users would only be using `service_account_creds`
- Some where in schema generator, if a schema is a list, that triggers app script, which would need to be deprecated
- there's also template micro that is hidden in the `master_template` (refer to `master_template_id`). We would need to remove that too 

**Describe the solution you'd like**

**How important is this feature?** Select from the options below:
• 🌗 Medium - can do work without it; but it's important (e.g. to save time or for convenience)

**When will use cases depending on this become relevant?** Select from the options below:
• Short-term - 2-4 weeks

**Additional context**

**Comment by linglp** [comment]
checking how app script authentication works (figure out an alternative way of authentication)

**Comment by linglp** [comment]
@milen-sage one more clarification question: if we are deprecating multi-select, can I assume that the column that previously requires multi-select would turn into a free text column that doesn't have any validation rules associated with it? For example, in the spreadsheet generated by our example data model [here](https://docs.google.com/spreadsheets/d/1edlBA41uHLeEGcEr6BxMfXnTjDYaBTdfA6HfGpX1Mfo/edit#gid=0), "FamilyHistory" would look like "Year of Birth"?

**Comment by milen-sage** [comment]
Yes - the attributes with multi-select would become free text. Perhaps we can change the tooltip to say something along the lines of "Please enter applicable comma-separated items selected from the set of allowable terms for this attribute. See our data standards for allowable terms."
